server/config/passport.js

```diff
@@ -0,0 +1,24 @@
+const { Strategy: JwtStrategy, ExtractJwt } = require("passport-jwt");
+
+const secret = require("./config").secretOrKey;
+const User = require("../models/user");
+
+exports = passport => {
+  passport.use(
+    new JwtStrategy(
+      {
+        secretOrKey: secret,
+        jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken()
+      },
+      (payload, done) => {
+        User.findById(payload.id).then(user => {
+          if (!user) {
+            done(null, false);
+          } else {
+            done(null, user);
```

Comment:
perfect - now the team should know @airensurzyn @umbauk  - that if you are using this middleware, you have access to the user using req.user

I don't want to see another fetch of the user when this middleware is being used